### ISSUE TITLE
Add document vehicle to issue

### ISSUE THREAD

**Issue by PaulCongdon** [jlm/maint#60]
While we have the 'draft with fix' field, this is helpful for finding the specific fix, once it is done.  It would be nice to have another field that can be used to associate the issue with a project even before it is fixed.   This would help the maintenance chair identify what editors should be contacted about specific issue, or what comments should be made against drafts by those editors to assure the issue gets addressed.
